defactor-com/ui-kit
# feat(LendBorrow): update component params and styles

## Improve LendBorrow component params and styles

### What does this PR do?

- Resolve #254 

### Steps to test

1. Run project `yarn storybook`
2. Check changes with new param `SelectedPool`

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/0331b61a-12b8-4ff6-9e09-0e1ee6bbde6c)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/65967e6b-4dd7-4162-a9d9-7500c64c5dd3)


#### CheckList
